pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -139,6 +139,12 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 		}
 	}
 
+	// ----------------------------TODO: remove these debug logs afterwards --------------------------------------
```

Comment:
If you wanted to keep these messages long-term, you could use log.V(some_high-ish_number).Info(...). That way you could switch them on in a given cluster without a code change in the future.